IntelliTect.TestTools.Data.Test/OtherSampleDbContext.cs

```diff
@@ -8,4 +8,14 @@ public class OtherSampleDbContext : DbContext
 
     public OtherSampleDbContext(DbContextOptions<OtherSampleDbContext> options) : base(options)
     { }
+}
+
+public class OtherSampleDbContextCopy : DbContext 
```

Comment:
This should probably be renamed to something else.
Should it be using `IQueryable`